pkg/apis/serving/v1/configuration_validation.go

```diff
@@ -36,7 +37,7 @@ func (c *Configuration) Validate(ctx context.Context) (errs *apis.FieldError) {
 		errs = errs.ViaField("metadata")
 
 		ctx = apis.WithinParent(ctx, c.ObjectMeta)
-		errs = errs.Also(c.Spec.Validate(apis.WithinSpec(ctx)).ViaField("spec"))
+		errs = errs.Also(c.Spec.Validate(helpers.WithAnnotations(apis.WithinSpec(ctx), c.Annotations)).ViaField("spec"))
```

Comment:
wonder if we could use [apis.ParentMeta](https://github.com/knative/serving/blob/278af32f24ce39f55b87c39d925435e04776559b/vendor/knative.dev/pkg/apis/contexts.go#L147) to avoid the extra helper pkg here

Reply:
Let me try it, should work I think.